### ISSUE TITLE
This makes it crash lol

### ISSUE THREAD

**Issue by peebo29** [notgoodusename/OsirisAndExtra#467]
This makes it crash lol
Its like displaying that the error is Abort()
![image](https://user-images.githubusercontent.com/83903616/210964762-79b304c1-d816-4c43-80b3-09e6d4a8fb46.png)

_Originally posted by @peebo29 in https://github.com/notgoodusename/OsirisAndExtra/issues/449#issuecomment-1373344508_

**Comment by peebo29** [comment]
Nah, cuz if there was any i would of pointed out the direct issue by now or fixed it. Couldn't find anything but just a Abort being called

**Comment by peebo29** [comment]
Im gonna try to debug it and see next time i crash within that error cuz im not really on it right now

**Comment by peebo29** [comment]
to be honest it must be something with the bullet_impact
I removed this one thing and it fixes itself (ghetto ass fix tho but it wont log the misses)

```
		interfaces->gameEventManager->addListener(&listener, "bullet_impact");
```

**Comment by peebo29** [comment]
Ok so no matter what i tried debugging and it just closed right away, so no debug info at all.

**Comment by Xsintashi** [comment]
try to break when abort() is called

**Comment by peebo29** [comment]
Ok i might of found why its breaking
The prediction reset might be a cause of it

```
void EnginePrediction::reset() noexcept
{
    localPlayerFlags = {};
    localPlayerVelocity = Vector{};
    netvarData = {};
    inPrediction = false;
}
```

**Comment by peebo29** [comment]
this might be a cause of a different issue while bypassing with an actual bypass and going into casual
